wikimedia/wikipedia-ios
# Change title to "Significant Updates"

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T266832

### Notes
* Updates the string

### Test Steps
1. Run app. Ensure the AaaLD interstitial in an article says "Significant Updates" for the header.

### Screenshots/Videos 
<img width="616" alt="Screenshot" src="https://user-images.githubusercontent.com/9295855/98054120-04cf0f00-1def-11eb-9d92-c9e0a388700c.png">
